locale/en/download/package-manager.md

```diff
@@ -92,6 +92,8 @@ dnf module install nodejs:12
 
 For CentOS/RHEL 7 Node.js is available via [Software Collections](https://www.softwarecollections.org/en/scls/?search=NodeJS).
 
+For Node.js > 12 is not available by this way because [this](https://bugzilla.redhat.com/show_bug.cgi?id=1728643) bug.
```

Comment:
+1 to what @richardlau says you can install Node.js 12, 14, and 16 on Fedora/CentOS/RHEL on RHEL 8 and 12 and 14 on RHEL 7.

Can you share the sequence of command lines that are not working for you and which version of the OS you are using.

Reply:
it can install but must supply profile named `default`.
so maybe we should change the command instead of add a note ?
pastebin [here](https://pastebin.com/iJsB9w0a)
my os is Fedora 32.

Reply:
@sakurayang I think the issue is that you are using Fedora 32. Based on an earlier issue I believe we have fixed it in later versions (ie with later versions, not sure but possibly 34 or later) you won't need to specify default.

Reply:
okay, I test again on both Fedora 32 and 33 with the same problem. But it solved in 35. 34 did not have default profile for nodejs16.
The reason I do this PR is that I found the problem unresolved when I followed #4392     .
Sorry for taking up your time.

Reply:
@sakurayang no problem hopefully as time moves forward all of the Fedora versions in people reach for will have the fix. Going to close the issue.